### ISSUE TITLE
Incorrect Text Size Segment Label

### ISSUE THREAD

**Issue by fredleblanc** [SiTaggart/AreMyColorsAccessible#656]
Sorry, not really sure what to call them. The left-most segment says “Small Text up to 18px,” but I believe it should say up to **24px**. (WCAG describes this as 18 points, which translates to 24px. Why a guidelines for the web would use a non-web-standard sizing like points is beyond me.)

This is somewhat backed up be the third column of “Large Type 24px and over.” The first column should describe text up to that line.

Side note: thanks for making this. It’s become the go-to tool for so many of the designers that I work with!

**Comment by SiTaggart** [comment]
Hiya @fredleblanc 

Yeah, it’s awkward. I think my original intent was to try to eliminate the sightless weird overlap between text and large text sizes. 

I figured by showing ratios in distinct size ranges with no over lap it would be easier for folks to understand the requirement. Text over 24px needs to be, bold text over 18px needs to be, text up to 18... 

I think since then the language in the criteria has been tweaked, though the rule not changed, which makes things a little clearer in my head and I could probably follow those definitions:

All text should be 4.5:1, except text over 24px and bold text over 18px which should be 3:1. 

I can probably just remove the text on the first contrast result block
